### PR TITLE
Provision GHC that corresponds to stack LTS

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

**Comment by jmitchell** [comment]
The CI job timed out. Can you ensure the CI script passes locally before merging?

File: src/Stack2nix/Util.hs [hunk 1]
```diff
@@ -58,16 +59,21 @@ logDebug args msg
...
-          hPutStrLn stderr $ err2
-          path <- getEnv "PATH"
-          setEnv "PATH" (path ++ ":" ++ unpack (strip (pack stdout)) ++ "/bin")
-        ExitFailure _ -> error $ nixAttr ++ " failed to build via nix"
+    Nothing -> ensureExecutable nixAttr
+
+-- given nixAttr, build it and add $out/bin to $PATH
+ensureExecutable :: String -> IO ()
+ensureExecutable nixAttr = do
+  (exitcode2, stdout, err2) <- readProcessWithExitCode "nix-build" ["-A", nixAttr, "<nixpkgs>"] mempty
```
Comment (puffnfresh) [inline src/Stack2nix/Util.hs]: Should this have `--no-build-output`?

Comment (domenkozar) [reply src/Stack2nix/Util.hs]: Thanks!

**Comment by domenkozar** [comment]
@jmitchell turns out CI failed due to stack2nix.nix being outdated.

**Comment by domenkozar** [comment]
It works now - but very close to travis-ci time limit.

**Pull request merged**